tests/lib/c_lib/strtoll/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
```

Comment:
I think you'll need to mirror the testcase.yaml file in ../strerror to add explicit tests for minimal libc. You should also add tests for picolibc (and newlib, but that won't matter after sdk 18, so meh?) to make sure they all work as expected.